en/admin/admin_utils.rst

```diff
@@ -290,6 +291,10 @@ The following example shows how to create a database, with additional volumes, i
     cubrid addvoldb -S -n cubriddb_DATA01 --db-volume-size=512M cubriddb
     cubrid addvoldb -S -p temp -n cubriddb_TEMP01 --db-volume-size=512M cubriddb
 
+.. note:: **Creating a database using an existing key file**
+
+    When the database is created, a key file is created together by default. If you want to use an existing key file when creating a database, first copy the key file with the name <database-name>_keys. Afterward, specify the directory path of the copied key file by the **ted_keys_file_path** system parameter, and create a database through the **createdb** utility. For more information on the TDE key file, see :ref:`tde-file-based-key`.
```

Comment:
```suggestion
    When the database is created, a key file is created together by default. If you want to use an existing key file when creating a database, first copy the key file with the name <database-name>_keys. Afterward, specify the directory path of the copied key file by the system parameter (**ted_keys_file_path**) and create a database through the **createdb** utility. For more information on the TDE key file, see :ref:`tde-file-based-key`.
```

- comment: sentence maybe too long or hard to read, consider rephrase it.

Reply:
How about this?
- I removed parentheses you suggetsed for consistency with others
```suggestion
    When the database is created, a key file is created together by default. If you want to use an existing key file when creating a database:
#.  Copy the key file with the name <database-name>_keys.
#.  Specify the directory path of the copied key file by the system parameter **tde_keys_file_path**.
#.  Create a database by using the **createdb** utility.

    For more information on the TDE key file, see :ref:`tde-file-based-key`.
```

Reply:
good!